### PR TITLE
Remove ERRDBG.

### PR DESCRIPTION
**Pull request by haydenroche5**
ERRDBG is a macro that's used to conditionally compile in various error logs. It's not used consistently, and we also already support log levels and NOTE_NODEBUG, which are more than enough to control the desired level of logging at compile-time.

### CODE REVIEW

File: n_serial.c [hunk 1]
```diff
@@ -93,9 +89,7 @@ const char *serialNoteTransaction(const char *request, size_t reqLen, char **res
         const char *err = serialChunkedReceive((uint8_t *)(jsonbuf + jsonbufLen), &jsonbufAvailLen, true, (CARD_INTRA_TRANSACTION_TIMEOUT_SEC * 1000), &available);
         if (err) {
             _Free(jsonbuf);
-#ifdef ERRDBG
-            NOTE_C_LOG_ERROR(ERRSTR("error occured during receive", c_iobad));
-#endif
+            NOTE_C_LOG_DEBUG(ERRSTR("error occured during receive", c_iobad));
```
Comment (haydenroche5) [inline n_serial.c:92]: @zfields Ping me if you want me to explain why the change to NOTE_C_LOG_DEBUG from NOTE_C_LOG_ERROR. I did that in a few places. Easier to explain verbally.

File: n_i2c.c [hunk 2]
```diff
@@ -58,18 +58,13 @@ NOTE_C_STATIC const char * i2cNoteQueryLength(uint32_t * available,
...
             NOTE_C_LOG_ERROR(err);
-#endif
             return err;
         }
 
         // If we've timed out, return an error
         if (timeoutMs && _GetMs() - startMs >= timeoutMs) {
             const char *err = ERRSTR("timeout: no response from Notecard {io}", c_iotimeout);
-#ifdef ERRDBG
-            NOTE_C_LOG_ERROR(err);
```
Comment (zfields) [inline n_i2c.c:71]: Did you intend to remove the actual log as well?

File: n_i2c.c [hunk 3]
```diff
@@ -414,11 +399,9 @@ const char *i2cChunkedReceive(uint8_t *buffer, uint32_t *size, bool delay, uint3
         // Exit on timeout
         if (timeoutMs && (_GetMs() - startMs >= timeoutMs)) {
             *size = received;
-#ifdef ERRDBG
             if (received) {
-                NOTE_C_LOG_ERROR(ERRSTR("received only partial reply before timeout", c_iobad));
+                NOTE_C_LOG_DEBUG(ERRSTR("received only partial reply before timeout", c_iobad));
```
Comment (zfields) [inline n_i2c.c:403]: Why the downgrade from `ERROR` to `DEBUG`?

IIRC, this means the response is invalid, and the transaction must be resubmitted.

I understand downgrading if it is in a retry loop, so long as it is eventually reported as an error once the retries have been exhausted.

File: n_serial.c (hunk 1 above)
Comment (zfields) [reply n_serial.c:92]: Ha! Okay, sure. You can disregard my previous note, I was working my way down before I saw this message.

File: n_serial.c [hunk 4]
```diff
@@ -190,11 +182,7 @@ bool serialNoteReset(void)
             break;
         }
 
-#ifdef ERRDBG
-        NOTE_C_LOG_ERROR(somethingFound ? ERRSTR("unrecognized data from notecard", c_iobad) : ERRSTR("notecard not responding", c_iobad));
```
Comment (zfields) [inline n_serial.c:194]: Seems like we should keep this message instead of the other one.

File: n_serial.c [hunk 5]
```diff
@@ -190,11 +182,7 @@ bool serialNoteReset(void)
             break;
         }
 
-#ifdef ERRDBG
-        NOTE_C_LOG_ERROR(somethingFound ? ERRSTR("unrecognized data from notecard", c_iobad) : ERRSTR("notecard not responding", c_iobad));
-#else
         NOTE_C_LOG_ERROR(ERRSTR("notecard not responding", c_iobad));
```
Comment (zfields) [inline n_serial.c:185]: Swap this one out for the preceding message.

**Review by zfields — APPROVED** [review]

**Review by zfields — APPROVED** [review]
